results/cmd/api/main_test.go

```diff
@@ -63,3 +63,33 @@ func setupTestDB(dbName string, t *testing.T) (*sql.DB, error) {
 	}
 	return db, nil
 }
+
+func TestGetTaskRun(t *testing.T) {
+	// Create a temporay database
+	testDB, err := setupTestDB("testdb", t)
+	if err != nil {
+		t.Fatalf("failed to create temp file for db: %v", err)
+	}
+
+	// connect to fake server and do testing
+	srv := &server{db: testDB}
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+	defer cancel()
+	objectmeta := &pb.ObjectMeta{Uid: "31415926", Name: "mytaskrun", Namespace: "default"}
+	r, err := srv.CreateTaskRun(ctx, &pb.CreateTaskRunRequest{TaskRun: &pb.TaskRun{ApiVersion: "v1beta1", Metadata: objectmeta}})
+	if err != nil {
+		t.Fatalf("could not create taskrun: %v", err)
+	}
+	t.Logf("Created taskrun: %s", r.String())
+
+	// Contact the server and get inserted data into database.
+	ctx, cancel = context.WithTimeout(context.Background(), time.Second)
+	defer cancel()
+
+	r, err = srv.GetTaskRun(ctx, &pb.GetTaskRunRequest{Uid: "123459"})
```

Comment:
Where is the value "123459" coming from? Is this deterministic?

Reply:
Shouldn't we be using r.GetUid() instead?

Reply:
Ah, I think this is the `uid INTEGER` field. We shouldn't key on this, because users have no way of knowing what this value is.

I'm also forgetting why we have both `uid` and `logid` can you add a comments to the schema definition to document why we have each field?

Reply:
You mean we shouldn't use uid to do query in the test? for now, to keep we get a unique taskrun, I just temporarily used it.
I've added why we have logid and uid  in results.sql